### PR TITLE
Create splash screen

### PR DESCRIPTION
**Pull request by TheNachi**
## What does this PR do?
Merges working Splash Screen to develop

## Description of Task to be completed?
Setup a stack navigator
Create a the splash screen component as the first component on the stack
Create an empty Login component  as the second component on the stack
Navigate to the Login after a setTimeOut

## How should this be manually tested?
Clone the app and try it out

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
#156042473 Splash screen 

## Screenshots (if appropriate)
N/A

## Questions:
N/A

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+// import { StyleSheet, Text, View } from 'react-native';
```
Comment (GbengaOshinaga) [inline App.js]: Remove commented lines

File: App.js [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
+import PrimaryNavigation from './src/PrimaryNavigation';
 
 export default class App extends React.Component {
   render() {
```
Comment (GbengaOshinaga) [inline App.js]: Add JSDoc to classes, methods and class properties

File: src/SplashScreen.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+export default class SplashScene extends Component {
+  componentDidMount() {
+    setTimeout(() => {
+      this.props.navigation.navigate('loginScreen');
+    }, 4000);
+  }
+
+  render() {
+    return (
+            <View style={styles.wrapper}>
```
Comment (GbengaOshinaga) [inline src/SplashScreen.js:19]: Use standard formatting for line spaces

File: src/SplashScreen.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                </View>
+                <Text style={styles.text}>
+                    All tribes... Assemble!
+                </Text>
+            </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (GbengaOshinaga) [inline src/SplashScreen.js:63]: Extract stylesheet into a separate file

**Review by ighotouch — APPROVED** [review]
This looks good

**Pull request merged**